crates/external_attestation/coordinator/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "external_attestation"
+version = "0.0.1"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+name = "external_attestation"
+
+[dependencies]
+hdk = { workspace = true }
+serde = { workspace = true }
+holoom_types = { workspace = true }
+shared_utils = { workspace = true }
+external_attestation_integrity = { workspace = true } 
+
+[target.'cfg(not(target_arch = "wasm32"))'.dev-dependencies] #avoids mio bug
+holochain = { workspace = true, default-features = false, features = ["test_utils"] }
+tokio = "1.38.0"
```

Comment:
Theses look like sweettest dependencies. Why are they here?

Reply:
yes for sweettests... every zome has its own tests directory